packages/components/pie-icon-button/README.md

```diff
@@ -8,37 +8,137 @@
   </a>
 </p>
 
+# Table of Contents
+
+1. [Introduction](#pie-icon-button)
+2. [Local Development](#local-development)
+3. [Props](#props)
+4. [Events](#events)
+   - [HTML example](#html)
+   - [Vue example (using Nuxt 3)](#vue-templates-using-nuxt-3)
+   - [React example (using Next 13)](#react-templates-using-next-13)
+5. [Testing](#testing)
+   - [Browser Tests](#browser-tests)
+   - [Visual Tests](#visual-tests)
+
+
 ## `pie-icon-button`
 
 `pie-icon-button` is a Web Component built using the Lit library. It offers a simple and accessible icon button component for web applications. This component can be easily integrated into various frontend frameworks and customized through a set of properties.
 
+## Local development
+
+Install the dependencies from the root of the monorepo:
```

Comment:
Might be worth making "from the root" bold, and saying that it also applies to all of the commands below too

Reply:
Reworded this now and added bold-ness 😄